src/Mutate4l/Commands/PrePad.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Mutate4l.Cli;
+using Mutate4l.Core;
+using Mutate4l.Utility;
+
+namespace Mutate4l.Commands
+{
+    // Extracts a region of the current clips, effectively cropping them. If two params, start - duration is used, otherwise 0 - duration.
+    public class PrePadOptions
+    {
+        [OptionInfo(type: OptionType.Default, 1/2f)]
+        public decimal[] Lengths { get; set; } = { 1 };
+    }
+
+    // # desc: Crops a clip to the desired length, or within the desired region.
+    public static class PrePad

```

Comment:
I think this command could solve more use cases if it also included a mechanism to set a desired length of the entire clip, where this command would then figure out if it needed to simply crop the clip (i.e. the original clip is too long, or add padding in order to make it the same as the length specified). It could be made into a more general command called Padding, where by default it would add padding as it does currently, but an additional option could make it behave like described above, for instance -fit or something like that.

Reply:
In the case of cropping, it could simply use the already existing crop command to achieve this.

Reply:
I take that on board.  so perhaps it should have the decimal list as arguments then.

Shall digest